ci-operator/step-registry/ipi/install/powervs/install/ipi-install-powervs-install-commands.sh

```diff
@@ -223,7 +223,7 @@ function install_required_tools() {
   export HOME=/tmp
 
   if [ ! -f /tmp/IBM_CLOUD_CLI_amd64.tar.gz ]; then
-    curl --output /tmp/IBM_CLOUD_CLI_amd64.tar.gz https://download.clis.cloud.ibm.com/ibm-cloud-cli/2.22.1/IBM_Cloud_CLI_2.22.1_amd64.tar.gz
+    curl --output /tmp/IBM_CLOUD_CLI_amd64.tar.gz https://download.clis.cloud.ibm.com/ibm-cloud-cli/2.24.0/IBM_Cloud_CLI_2.24.0_amd64.tar.gz
```

Comment:
Is there a way to grab this more programmatically?  a `latest` variant or something?  Unless we need this specific version, I'd rather it be something we don't update manually.

Reply:
Discussed via Slack

Reply:
It sounds like
```
curl -fsSL https://clis.cloud.ibm.com/install/linux | sh
```
downloads the latest.  But if the website is hacked, then a malicious script could be executed in CI.  I guess the same could be said for binaries.

Reply:
Pinning for stability is a fine answer... Do we think it would be more valuable to make the version a variable?  Configure it at the job level rather than in the script?  If we don't expect this to change all that often then no worries.

Reply:
That would be a good future task @mjturek should accept to get to know how CI and ci-operator works

Reply:
Yeah that'd be fine to tackle later, then.  Sure.